docker/puppetdb/docker-entrypoint.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
 
+master_running() {
+    curl -sf http://master:8140/production/status/test \
```

Comment:
should this be `$PUPPETSERVER_HOSTNAME` instead of `master`?

Reply:
Ugh, yes, it should. Nice catch. Updating the PR.

Reply:
Ugh, yes it should. Nice catch! Updating the PR.